components/merkledb/src/views/metadata.rs

```diff
@@ -241,9 +242,9 @@ impl<V> IndexMetadata<V> {
     }
 
     /// Returns a globally unique numeric index identifier.
-    /// MerkleDB assigns a unique numeric ID for each fully-qualified index name.
+    /// `MerkleDB` assigns a unique numeric ID for each fully-qualified index name.
```

Comment:
IMO, this change is confusing; I'm expecting code-formatted bits, which look like type names, to correspond to the type names.